### PR TITLE
Repairer points to redis server

### PR DESCRIPTION
**Pull request by aleitner**

### CODE REVIEW

File: pkg/datarepair/repairer/repairer.go [hunk 1]
```diff
@@ -27,8 +33,9 @@ type Repairer interface {
 
 // Config contains configurable values for repairer
 type Config struct {
-	// queueAddress string `help:"data repair queue address" default:"localhost:7779"`
-	maxRepair int `help:"maximum segments that can be repaired concurrently" default:"100"`
+	queueAddress string `help:"data repair queue address" default:"localhost:6379"`
+	queuePass    string `help:"data repair queue password" default:""`
```
Comment (egonelbre) [inline pkg/datarepair/repairer/repairer.go]: Maybe pass in similarly how overlay and pointerdb does it. That way you don't have to have separate password field and can pass in different implementations (later). Then you can specify it as `redis://localhost:6379?db=5&password=123`.

See:
https://github.com/storj/storj/blob/b6b6111173e524ec55d3abc699c37ef816d51786/pkg/pointerdb/config.go#L26
https://github.com/storj/storj/blob/b6b6111173e524ec55d3abc699c37ef816d51786/pkg/overlay/config.go#L46

File: storage/redis/client.go [hunk 2]
```diff
@@ -44,6 +47,23 @@ func NewClient(address, password string, db int) (*Client, error) {
 	return client, nil
 }
 
+// NewClientFrom returns a configured Client instance from a redis address, verifying a successful connection to redis
+func NewClientFrom(address string) (*Client, error) {
+	redisurl, err := utils.ParseURL(address)
```
[the remark this answers is not in the document]
Comment (aleitner) [reply storage/redis/client.go]: Do you just want me to change variable name, and comment?

[the remark this answers is not in the document]
Comment (egonelbre) [reply storage/redis/client.go]: Check the schema and if it isn't "redis" then return an error.

File: storage/redis/client.go [hunk 3]
```diff
@@ -54,6 +55,10 @@ func NewClientFrom(address string) (*Client, error) {
 		return nil, err
 	}
 
+	if strings.Compare("redis", redisurl.Scheme) != 0 {
```
[the remark this answers is not in the document]
Comment (aleitner) [reply storage/redis/client.go]: I code in C too much

**Pull request merged**